atm.js

```diff
@@ -1,3 +1,37 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+//These are wrong, should probably be parseInt's
```

Comment:
Are you referring to your selectors in `left`? They look fine to me. It's useful to define references to these DOM objects.